### PR TITLE
dolt-clone: add page

### PR DESCRIPTION
**Pull request by wenz**
For #5471

- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/dolt-clone.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# dolt clone
+
+> Clone a repository into a new directory.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-clone>.
+
+- Clone a repository into a new directory (defaults to the repository name):
+
+`dolt clone {{repository_url}} {{directory_name}}`
+
+- Clone a repository and add a remote (defaults to origin):
```
Comment (navarroaxel) [inline pages/common/dolt-clone.md:10]:
```suggestion
- Clone an existing repository and add a specific remote (defaults to origin):
```

File: pages/common/dolt-clone.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# dolt clone
+
+> Clone a repository into a new directory.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-clone>.
+
+- Clone a repository into a new directory (defaults to the repository name):
```
Comment (navarroaxel) [inline pages/common/dolt-clone.md:6]:
```suggestion
- Clone an existing repository into a specific directory (defaults to the repository name):
```

File: pages/common/dolt-clone.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`dolt clone {{repository_url}} {{directory_name}}`
```
Comment (navarroaxel) [inline pages/common/dolt-clone.md:8]: Git supports a path, I think Dolt too
```suggestion
`dolt clone {{repository_url}} {{path/to/directory}}`
```

File: pages/common/dolt-clone.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Clone a repository and add a remote (defaults to origin):
+
+`dolt clone --remote {{remote_name}} {{repository_url}}`
+
+- Clone a branch in the repository (defaults to all branches):
```
Comment (navarroaxel) [inline pages/common/dolt-clone.md:14]: Based on the `git-clone` page
```suggestion
- Clone an existing repository only fetching a specific branch (defaults to all branches):
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by wenz** [comment]
Thank you for the suggestions, committed them!

**Review by marchersimon — APPROVED** [review]
Thanks @wenz

File: pages/common/dolt-clone.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`dolt clone --aws-creds-file {{creds_file}} {{repository_url}}`
+
+- Clone a repository, using an AWS credentials profile (uses the default profile if none is provided):
+
+`dolt clone --aws-creds-profile {{profile_name}} {{repository_url}}`
+
+- Clone a repository, using an AWS credentials type:
+
+`dolt clone --aws-creds-type {{creds_type}} {{repository_url}}`
```
Comment (navarroaxel) [inline pages/common/dolt-clone.md:32]:
```suggestion
`dolt clone --aws-creds-type {{credentials_type}} {{repository_url}}`
```

File: pages/common/dolt-clone.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`dolt clone --branch {{branch_name}} {{repository_url}}`
+
+- Clone a repository, using an AWS region (uses the profile's default region if none is provided):
+
+`dolt clone --aws-region {{region_name}} {{repository_url}}`
+
+- Clone a repository, using an AWS credentials file:
+
+`dolt clone --aws-creds-file {{creds_file}} {{repository_url}}`
```
Comment (navarroaxel) [inline pages/common/dolt-clone.md:24]:
```suggestion
`dolt clone --aws-creds-file {{credentials_file}} {{repository_url}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**